### ISSUE TITLE
loss explode at about 150 steps

### ISSUE THREAD

**Issue by 915288938lx** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#437]
tensorflow 1.10.0 , I labeled the pictures of your tutorial both train part and test part , when i start training, the loss value begin to explode when it comes to about 150 steps with loss value about 0.3, and when the train schedule goes to 200 steps, the loss value increase to 144812346.. 
![c0ffed7717c752740887973b734dcc3](https://user-images.githubusercontent.com/24892574/73813406-db129880-481a-11ea-82aa-5f9f8928f247.png)
![f29c01470c2dd545362a548911d7fe2](https://user-images.githubusercontent.com/24892574/73813410-dc43c580-481a-11ea-9dc3-f1f2f3730891.png)
![7eab2b6b2b1401fff134f4c311fcbdb](https://user-images.githubusercontent.com/24892574/73813414-df3eb600-481a-11ea-8026-5f1980987563.png)
![6ac65e695034c318b6fb4dad090ef5f](https://user-images.githubusercontent.com/24892574/73813419-e1087980-481a-11ea-9ef6-b1fcb82b584d.png)

**Comment by EdjeElectronics** [comment]
This happened to me recently, I fixed it by making sure that "num_classes" is set to the correct value in the .config file. Example: if you are training your model to detect 3 different classes, set "num_classes: 3" in faster_rcnn_inception_v2_pets.config .

Please re-open this issue if that doesn't fix it!
